### ISSUE TITLE
Add Helpful Readme

### ISSUE THREAD

**Issue by cjacoby** [constd/auem#3]
Add a readme which includes how to install the environment.

**Comment by constd** [comment]
True, we also probably need to start describing what the repository is about. How to use, etc.
Do you think we should be doing this all via READMEs or should we be looking at [Sphinx](http://www.sphinx-doc.org/en/stable/) or similar?

**Comment by cjacoby** [comment]
Let's start with the readme and expand to sphinx as it becomes big enough.
